cranelift/codegen/Cargo.toml

```diff
@@ -23,7 +23,7 @@ serde = { version = "1.0.94", features = ["derive"], optional = true }
 bincode = { version = "1.2.1", optional = true }
 gimli = { version = "0.26.0", default-features = false, features = ["write"], optional = true }
 smallvec = { version = "1.6.1" }
-regalloc2 = { version = "0.2.2", features = ["checker"] }
+regalloc2 = { version = "0.2.2", path = "../../../regalloc2/", features = ["checker", "trace-log"] }
```

Comment:
If you've got a git branch/rev to point this to that might be good to use here to turn the crank on Wasmtime's CI to ensure it's green ahead of time.